tests/utils.go

```diff
@@ -2001,9 +2001,13 @@ func GetGuestAgentUserData() string {
 
 	ipv6UserDataString := ""
 	if netutils.IsIPv6String(dnsServerIP) {
-		ipv6UserDataString = fmt.Sprintf(`sudo ip -6 addr add fd10:0:2::2/120 dev eth0
-                             sudo ip -6 route add default via fd10:0:2::1 src fd10:0:2::2
-                             echo "nameserver %s" >> /etc/resolv.conf`, dnsServerIP)
+		ipv6UserDataString = fmt.Sprintf(`sudo ip -6 addr add %s dev eth0
```

Comment:
I would reconsider touching a module that is over 4500 lies of code.
If you want to refactor something, then please take things out to a more focused module and help future you (and current me) to understand what `utils` mean.

Reply:
I removed this commit, I will continue with this in a new PR.